.eslint-dictionary.json

```diff
@@ -280,6 +290,7 @@
   "readline",
   "readme",
   "readonly",
+  "realpath",
```

Comment:
nit: can this be replaced in code with `realPath`

Reply:
realpath is a method on the fs library